lib/oxidized/model/comnet.rb

```diff
@@ -0,0 +1,44 @@
+class Comnet < Oxidized::Model
```

Comment:
I've just had a look and Comnet is the vendor and provides lots of products. The model should be named after the firmware the devices run like IOS, JUNOS, etc.

Can you change the name to match the firmware, if they don't provide firmware versions then it will need to at least be a name to cover the models they support.

Reply:
Ok but if this model doesn't cover the entire comnet hardware range of devices (not just switches) then we should consider a better name imho.

Reply:
@jaylik Any update on this?